app/client/contract.ts

```diff
@@ -560,8 +560,12 @@ export const definition = {
         .boolean()
         .default(true)
         .describe(
-          "Allow the run to be continued with follow-up messages / message edits",
+          "Allow the run to be continued with follow-up messages / message edits"
         ),
+      enableResultGrounding: z
```

Comment:
/nit Should probably name this `resultGrounding` for consistency with other options (or rename them).

We can do this once the feature is not flagged though.